### DISCUSSION TITLE
COVID-19 Rapid Response Research screen time questions from COVID surveys 4-7

### DISCUSSION THREAD

**Discussion by carly-gray-19** [now-i-know-my-abcd/docs#98] [Where in the World is…? (Help)] [answered]
Hi all,
I'm trying to find information about what the response options were for the questions about screen time during COVID-19 in surveys 4-7 of the COVID-19 Rapid Response Research surveys. According to the [release notes](https://wiki.abcdstudy.org/release-notes/substudies/covid-19.html#screen-media-use-for-school-socializing-and-other-reasons) and the data dictionary,
> For the first 3 Surveys released the response answers were 0 minutes, 15 minutes, 30 minutes, 45 mins, 1 hour, 1.5 hours, 2 hours, 2.5 hours, and then each whole hour through 24 hours. 

However, I'm unable to find any information about what the response options change _to_ in surveys 4-7. It appears that the range of options was 0-14 from my exploration of the data, so maybe it's the same set of answers but with a maximum of 9+ hours/day?
Any help is appreciated!

**Comment by stapert** [comment]
Hi, Carly!

I looked back at our forms, and I believe in Survey 4, response options were 0 minutes, 15 minutes, 30 minutes, 45 mins, 1 hour, 1.5 hours, 2 hours, 2.5 hours, and then each whole hour through 24 hours. 

For Surveys 5, 6, and 7, so that the response options could be replied to via a mobile device, we truncated response options to:
15 minutes
30 minutes
45 minutes
1 hour
1.5 hours
2 hours
2.5 hours
3 hours
4 hours
5 hours
6 hours
7 hours
8 hours
9 hours or more

Does this help?

**Comment by stapert** [comment, marked as answer]
Hi, Carly!

I looked back at our forms, and I believe in Survey 4, response options were 0 minutes, 15 minutes, 30 minutes, 45 mins, 1 hour, 1.5 hours, 2 hours, 2.5 hours, and then each whole hour through 24 hours.

For Surveys 5, 6, and 7, so that the response options could be replied to via a mobile device, we truncated response options to:
15 minutes
30 minutes
45 minutes
1 hour
1.5 hours
2 hours
2.5 hours
3 hours
4 hours
5 hours
6 hours
7 hours
8 hours
9 hours or more

**Reply by carly-gray-19** [reply]
This is exactly what I was looking for, thank you so much!
